modin/backends/pandas/query_compiler.py

```diff
@@ -1313,6 +1299,28 @@ def is_tree_like_or_1d(calc_index, valid_index):
         result = self.__constructor__(new_modin_frame)
 
         def compute_index(index, columns, consider_index=True, consider_columns=True):
+            """
+            Compute new index for the unstacked frame.
```

Comment:
what does "unstacked" mean here?

Reply:
this function is nested in the `unstack` method, and so compute new indices for the resulted (unstacked) frame